### ISSUE TITLE
DB/Command: server restart/idle restart

### ISSUE THREAD

**Issue by BarbzYHOOL** [azerothcore/azerothcore-wotlk#531]
MINOR ISSUE

**Description**: These commands never worked for me so I decided to open an issue.
It's not really a problem when you have a restart script but well, it's not good for beginners or for ease of use.

**Current behaviour**: If you type these commands, it only shutdowns the server, but never restarts.

**Expected behaviour**: It should restart as noted in the description and the name.

**Steps to reproduce the problem**:

1. Type this in the console : .server restart
2. It won't restart

**Branch(es)**: master

**AC hash/commit**:  1521451529c1cae72a19248068da041c5bb2efbf

**Operating system**:  debian 8

**Modules**:  Anticheat

**Comment by Kizarr** [comment]
Confirmed I have same issue it also a bug on trinitycore.

**Comment by Viste** [comment]
why for me its work i use debug modeog build

**Comment by poszer** [comment]
Without another program/script to restart it, it is very risky to touch this. If you take a look in shutdown/restart command, only the difference between them is shutdown mask. I'll suggest to close this - Wont fix

**Comment by gengarshadowball** [comment]
Can anyone give feedback on this? If it doesn't work I can make a PR to fix or a PR to remove it in case this isn't a wanted feature.

**Comment by Viste** [comment]
the command never works ) WoWD never support restart only start/stop

**Comment by BarbzYHOOL** [comment]
ok so we can remove it?

**Comment by Viste** [comment]
yes ofc or rewrite all related code )))))))

**Comment by BarbzYHOOL** [comment]
thanks to misleading advices here, we discovered it must not be removed :D

the only thing that could be done is either to fix the issue, or to edit the command description

also nobody uses .server idlerestart apparently

**Comment by Zoidwaffle** [comment]
Hm, this is kinda interesting... Talking from the perspective of a Linux user, it's weird to have a program being able to restart itself as I would consider this a violation of standard practices for services on a server.

Maybe it's just me - and I would be fine with a "reload" command that can be initiated from within the service itself. And the "idlerestart" mentioned sounds like a great feature, except it should be handled from outside with a "idleshutdown" command and then a "start" afterwards.

In short: A service restart is something I would expect to be triggered from outside as the PID should change.

YMMW and I only speak on behalf of myself.

**Comment by Kitzunu** [comment]
The goal of the command wasnt for it to automatically restart without an external script.

The only thing the command does is sending another exit code so that the script knows.
